python/ml4ir/applications/classification/config/parse_args.py

```diff
@@ -0,0 +1,31 @@
+from argparse import Namespace
+
+from ml4ir.base.config.keys import TFRecordTypeKey
+
+from ml4ir.applications.ranking.config.keys import LossKey, MetricKey
+from ml4ir.base.config.parse_args import RelevanceArgParser
+
+from typing import List
+
+
+class ClassificationArgParser(RelevanceArgParser):
+    """
+    States default arguments for classification model.
+    """
+    def set_default_args(self):
+        super().set_default_args()
+        self.set_defaults(
+            tfrecord_type=TFRecordTypeKey.EXAMPLE,
+            loss_key=LossKey.CATEGORICAL_CROSS_ENTROPY,
+            metrics_keys="['" + MetricKey.CATEGORICAL_ACCURACY + "'"
+                         + ", '" + MetricKey.PRECISION + "'"
+                         + "]",
+            monitor_metric=MetricKey.CATEGORICAL_ACCURACY,
+            monitor_mode="max",
+            max_sequence_size=25,
```

Comment:
I don't know why we need max_sequence_size here. It seems to be required?

Reply:
You don't. You can set it to None or 0 I think. This is only used for SequenceExample.

Reply:
do we not default to None already, @lastmansleeping ?  We shouldn't require this arg at all for Example-based setups.

Reply:
It is defaulted to 0. I don't understand the error that @Ullimague is facing.

Reply:
My bad, it was the group_metrics_min_queries line 26 that is required.